frontend/src/js/query-runner/DownloadResultsDropdownButton.tsx

```diff
@@ -105,7 +126,7 @@ const DownloadResultsDropdownButton = ({
       {!tiny && (
         <>
           <SxDownloadButton bgHover url={urlChoice}>
-            {endingChoice}
+            {urlChoice.label}
```

Comment:
Did you test what it looks like with longer labels? Is there a max length we should restrict this to?

Reply:
![image](https://user-images.githubusercontent.com/118362236/218537195-badc5efa-590b-456c-8c16-f4dc315735f0.png)
The content will expand to the left. I think that looks acceptable, since I believe extremely long labels will be rare. If you think that it is necessary, I can write a function that truncates the output

Reply:
The overlay content itself will be fine, but I'm asking about the button label itself. 

Because this is what it will look like

<img width="940" alt="Screenshot 2023-02-20 at 12 19 56" src="https://user-images.githubusercontent.com/1403093/220091626-abb4cfa8-dc3e-4dac-8808-4c1db48721a2.png">

> I believe extremely long labels will be rare

Could you discuss with the team members what exactly they will look like?

If longer than 40 characters could happen, I'd say we have to add truncation

Reply:
That makes sense, I just added the truncation for it. Will truncate after 40 chars to 37 plus ...